2023-03-18 13:19:34,849 INFO    MainThread:25416 [wandb_setup.py:_flush():68] Configure stats pid to 25416
2023-03-18 13:19:34,849 INFO    MainThread:25416 [wandb_setup.py:_flush():68] Loading settings from C:\Users\canba\.config\wandb\settings
2023-03-18 13:19:34,849 INFO    MainThread:25416 [wandb_setup.py:_flush():68] Loading settings from C:\Users\canba\OneDrive\Masaüstü\OPFGNN\code\wandb\settings
2023-03-18 13:19:34,850 INFO    MainThread:25416 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-18 13:19:34,850 INFO    MainThread:25416 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-18 13:19:34,850 INFO    MainThread:25416 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\canba\OneDrive\Masaüstü\OPFGNN\code\wandb\run-20230318_131934-ilmn9zlz\logs\debug.log
2023-03-18 13:19:34,850 INFO    MainThread:25416 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\canba\OneDrive\Masaüstü\OPFGNN\code\wandb\run-20230318_131934-ilmn9zlz\logs\debug-internal.log
2023-03-18 13:19:34,850 INFO    MainThread:25416 [wandb_init.py:init():532] calling init triggers
2023-03-18 13:19:34,851 INFO    MainThread:25416 [wandb_init.py:init():539] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'architecture': 'Homo-GNN', 'num_layers': 4, 'dataset': '1-HV-mixed--0-no_sw', 'epochs': 400, 'activation': 'tanh', 'dropout': 0.0, 'in_channels': 4, 'output_channels': 4, 'hidden_channels': 186, 'channel type': 'TransConv', 'norm': 'BatchNorm'}
2023-03-18 13:19:34,851 INFO    MainThread:25416 [wandb_init.py:init():555] re-initializing run, found existing run on stack: p3i44a3p
2023-03-18 13:19:34,852 INFO    MainThread:25416 [wandb_run.py:_finish():1854] finishing run cbml/OPF-GNN-Supervised-Homo/p3i44a3p
2023-03-18 13:19:34,853 INFO    MainThread:25416 [jupyter.py:save_history():449] not saving jupyter history
2023-03-18 13:19:34,853 INFO    MainThread:25416 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-18 13:19:34,853 INFO    MainThread:25416 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-18 13:19:34,853 INFO    MainThread:25416 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-18 13:19:34,854 INFO    MainThread:25416 [wandb_run.py:_restore():2066] restore
2023-03-18 13:19:34,854 INFO    MainThread:25416 [wandb_run.py:_restore():2072] restore done
2023-03-18 13:19:39,272 INFO    MainThread:25416 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-18 13:19:39,272 INFO    MainThread:25416 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-18 13:19:39,277 INFO    MainThread:25416 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-18 13:19:39,281 INFO    MainThread:25416 [wandb_init.py:init():588] starting backend
2023-03-18 13:19:39,281 INFO    MainThread:25416 [wandb_init.py:init():592] setting up manager
2023-03-18 13:19:39,283 INFO    MainThread:25416 [backend.py:_multiprocessing_setup():108] multiprocessing start_methods=spawn, using: spawn
2023-03-18 13:19:39,285 INFO    MainThread:25416 [wandb_init.py:init():599] backend started and connected
2023-03-18 13:19:39,297 INFO    MainThread:25416 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-18 13:19:39,297 INFO    MainThread:25416 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-18 13:19:39,297 INFO    MainThread:25416 [wandb_init.py:init():687] updated telemetry
2023-03-18 13:19:39,344 INFO    MainThread:25416 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-18 13:19:40,791 INFO    MainThread:25416 [wandb_run.py:_on_init():2134] communicating current version
2023-03-18 13:19:41,050 INFO    MainThread:25416 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-18 13:19:41,051 INFO    MainThread:25416 [wandb_init.py:init():775] starting run threads in backend
2023-03-18 13:19:44,934 INFO    MainThread:25416 [wandb_run.py:_console_start():2114] atexit reg
2023-03-18 13:19:44,934 INFO    MainThread:25416 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-18 13:19:44,934 INFO    MainThread:25416 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-18 13:19:44,934 INFO    MainThread:25416 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-18 13:19:44,935 INFO    MainThread:25416 [wandb_init.py:init():817] run started, returning control to user process
2023-03-18 13:38:01,600 WARNING MsgRouterThr:25416 [router.py:message_loop():77] message_loop has been closed
